churaverse-plugins-client/src/synchroBreakPlugin/ui/descriptionWindow/descriptionWindow.ts

```diff
@@ -1,21 +1,30 @@
 import { DomManager, domLayerSetting } from 'churaverse-engine-client'
 import { DescriptionWindowComponent } from './component/DescriptionWindowComponent'
 import '@churaverse/game-plugin-client/gameUiManager'
-import { IDescriptionWindow } from '../../interface/IDescriptionWindow'
+import { IDescriptionText } from '../../interface/IDescriptionText'
 
-export class DescriptionWindow implements IDescriptionWindow {
+export class DescriptionWindow implements IDescriptionText {
   public element!: HTMLElement
   public visible: boolean = false
   private descriptionText: string = ''
+  private gameName: string = 'ゲーム'
+  private gameOwnerName: string = 'ゲームオーナー'
+
+  public setGameName(name: string): void {
+    this.gameName = name
+  }
+
+  public setGameOwnerName(name: string): void {
+    this.gameOwnerName = name
+  }
```

Comment:
こちら二つの関数が、それぞれ同じ箇所と一度しか呼び出しをしないとおもうので、一つの関数にまとめても良いとも思うのですがどうでしょうか？

```suggestion
public setGameBaseInfo(gameName: string, ownerName: string): void {
  this.gameName = gameName
  this.gameOwnerName = ownerName
}
```